httpserver/service_node.cpp

```diff
@@ -13,11 +13,13 @@
 #include <chrono>
 #include <fstream>
 #include <iomanip>
+#include <resolv.h>
```

Comment:
This is not portable, is it?

Reply:
After talking to Simon, plan is to leave windows support on TODO list